pom.xml

```diff
@@ -21,6 +21,10 @@
   <groupId>co.cask.hydrator</groupId>
   <artifactId>hydrator-plugins</artifactId>
   <version>1.2.0</version>
+  <name>Hydrator Plugin Collection</name>
+  <description>A collection of plugins for Cask Hydrator.</description>
+  <url>http://docs.cdap.io/cdap/current/en/included-applications/index.html</url>
```

Comment:
I just moved the url line from a few lines below.
Want me to change it to `https://github.com/caskdata/hydrator-plugins`?